go-indexer/indexer/indexer.go

```
@@ -36,25 +37,27 @@
 	healthServer   *health.Server
 }
 
-func New(cfg *config.Config) (*Indexer, error) {
+func New(ctx context.Context, cfg *config.Config) (*Indexer, error) {
```

golangci-lint (1.64.8)

SA4009: argument ctx is overwritten before first use

(staticcheck)